_analyzers/token-filters/keyword-marker.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: default
+title: Keyword marker
+parent: Token filters
+nav_order: 200
+---
+
+# Keyword marker token filter
+
+The `keyword_marker` token filter in OpenSearch is used to prevent certain tokens from being altered by stemmers or other filters. This ensures that specific words remain in their original form. `keyword_marker` token filter does this by marking tokens as `keywords` which prevents any stemming or other processing.
```

Comment:
```suggestion
The `keyword_marker` token filter is used to prevent certain tokens from being altered by stemmers or other filters. The `keyword_marker` token filter does this by marking the specified tokens as `keywords`, which prevents any stemming or other processing. This ensures that specific words remain in their original form. 
```